gradoop-flink/src/main/java/org/gradoop/flink/model/api/epgm/BaseGraphCollectionOperators.java

```diff
@@ -84,7 +84,7 @@
    * @param graphID graph identifier
    * @return base graph with given id or an empty base graph
    */
-  LG getGraph(final GradoopId graphID);
+  LG getGraph(GradoopId graphID);
```

Comment:
Does checkstyle really complain about this? Weird...

Reply:
It is a `RedundantModifier`. Final in interface method parameters does not mean anything. The implementer can still use non final parameters and change them. It could be used as hint for the implementer, but it is not enforced.